src/core/Makefile.am

```diff
@@ -81,6 +81,7 @@ libopenthread_a_SOURCES             = \
     thread/panid_query_server.cpp     \
     thread/thread_netif.cpp           \
     thread/thread_tlvs.cpp            \
+    utils/global_address.cpp          \
```

Comment:
The UniqueuIdentifier is just a random GUID. You can use this one I just created `{A0D2E990-DC5E-4B56-A552-888F97214A17}`